### ISSUE TITLE
About training data

### ISSUE THREAD

**Issue by XiaoqiangZhou** [owenzlz/SuperCAF#1]
Thanks for sharing such a great and meaningful work on ultra-high image inpainting. 

However, the current repo lacks data and codes for training. Will you share the 48229 training images and inpainting masks, and training codes? 

Thanks.
